deploy/pmm.yaml

```diff
@@ -0,0 +1,153 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: pmm
+  labels:
+    app.kubernetes.io/name: pmm
+    app.kubernetes.io/instance: pmm
+spec:
+  serviceName: monitoring-service
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: pmm
+      app.kubernetes.io/instance: pmm
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: pmm
+        app.kubernetes.io/instance: pmm
+    spec:
+      # uncomment service account when deploying pmm-rbac.yaml (should be deployed before this manifest)
+      #serviceAccountName: pmm-service-account
+      containers:
+        - name: pmm
+          image: "percona/pmm-server:2.27.0"
+          imagePullPolicy: Always
+
+          # environment passed to PMM, defined in configMap
+          envFrom:
+            - configMapRef:
+                name: pmm
+                optional: true
+
+          # additional secrets that could be used for Grafana iDP 
+          env:
+            - name: GF_AUTH_GENERIC_OAUTH_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: pmm-secret
+                  key: GF_AUTH_GENERIC_OAUTH_CLIENT_ID
+                  optional: true
+            - name: GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: pmm-secret
+                  key: GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET
+                  optional: true
+          ports:
+            - name: http
+              containerPort: 80
+            - name: https
+              containerPort: 443
+          readinessProbe:
+            httpGet:
+              path: /v1/readyz
+              port: http
+          volumeMounts:
+            - name: pmm-storage
+              mountPath: /srv
+      initContainers: # Execute a set of changes in persistent volume and ensure they are met on each deployment.
+        - name: pmm-init
+          image: "percona/pmm-server:2.27.0"
+          imagePullPolicy: Always
+          command: ["/bin/bash"]
+          args:
+            - "-c"
+            - |
```

Comment:
yeah, I thought it would be part of https://jira.percona.com/browse/PMM-8082 . But after investigation drop the idea because of big implementation.
As supervisord runs everything in parallel there is no way to have some init as a first task.

We can modify entry point, but it is ugly. We want to simplify entry point, not to further complicate it. It is essentially and extension to entry point here.